### PR TITLE
January heartbeat

### PR DESCRIPTION
**Pull request by jendefig**
January Heartbeat

**Publish Date**: 01-18-22
Cover image is a draft.  Should have final soon.  Making change this month to keep image clean (free from words) as the title is under it on the page.  Title and month to be added for socials.  Please review and give feedback on this new change as well. 

- [ ]  There are no grammatical errors, jargon, or obvious stylistical issues. (Reviewer)
- [x]  The flow and messaging corresponds to what has been defined in the Phase 3.
- [x]  Have images been run through tiny.png?
- [x]  Are all lines 80 characters or less?
- [x]  Are Oxford commas observed?
- [x]  Content Creator has done complete review for typos/language after taking break from content
- [x]  links checked in piece
- [x]  Have reviewed what the piece looks like on your own [machine using yarn develop](https://dvc.org/doc/user-guide/contributing/docs#development-environment)

### CODE REVIEW

**Comment by jendefig** [comment]
@julieg18 Need help.  For some reason the the blog piece isn't showing up in Heroku deployment.  What did I do wrong?  Can see on my computer in yarn develop.  I had a copy of it, deleted it and put the correct file name.  But for some reason it's just showing up with the last blog post in Heroku.

**Comment by jendefig** [comment]
> @julieg18 Need help. For some reason the the blog piece isn't showing up in Heroku deployment. What did I do wrong? Can see on my computer in yarn develop. I had a copy of it, deleted it and put the correct file name. But for some reason it's just showing up with the last blog post in Heroku.

Nevermind @julieg18!  Got it figured out!

File: content/blog/2022-1-18-January-22-heartbeat.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+
+[In Part 2](https://poatek.com/2021/12/29/mlops-a-complete-and-hands-on-introduction-part-2/)
+he delves into more detail and code on how to set up version control of
+everything with DVC and automation of experimentation and reporting with CML.
+Finally he uses FastAPI and Herouku for model serving and deployment. You can
+find all the scripts for the project in
+[this GitHub repository.](https://github.com/dsjardim/fraud-detection-mlops)
+
+<external-link 
+href="https:/https://poatek.com/2021/12/29/mlops-a-complete-and-hands-on-introduction-part-2/"
```
Comment (rogermparent) [inline content/blog/2022-1-18-January-22-heartbeat.md:65]: Small typo here with `https:/`

**Comment by rogermparent** [comment]
Looks like Medium doesn't play well with the link checker! I guess we'll need some extra configuration like with linkedin profiles.

File: content/blog/2022-1-18-January-22-heartbeat.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+training (the model continues training on new data) and moving from a manual
+process to a more automated one. Definitely worth a read and we believe DVC and
+CML can help you with your stateful training!
+
+She and her team are running a [survey](https://forms.gle/dDvgF7QgpPdvJE5b8) to
+better understand the adoption and challenges of real-time ML. We enourage your
+particpation!
+
+![Stateful Training](/uploads/images/2022-01-18/stateful-training.png '=800')
+_Chip Huyen's Statelyss vs.Stateful Training
```
Comment (MKhalusova) [inline content/blog/2022-1-18-January-22-heartbeat.md:221]: Statelyss -> Stateless

File: content/blog/2022-1-18-January-22-heartbeat.md [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+## Experiment Versioning piece in KDNuggets
+
+Our Senior Developer Advocate
+[**Maria Khalusova**](https://twitter.com/mariaKhalusova) wrote a tutorial piece
+on `exp init` and experiment versioning entitled
+[Versioning Machine Learning Experiments vs Tracking Them.](https://www.kdnuggets.com/2021/12/versioning-machine-learning-experiments-tracking.html)
+The command helps you quickly set up a pipeline and codify your experiments with
+all of the factors that contributed to each of them, including data, code,
+pipeline, model version and all hyperparameters. This is a step above other
+experiment tracking tools and enables you achieve true reproducibility.
```
Comment (MKhalusova) [inline content/blog/2022-1-18-January-22-heartbeat.md:259]: to achieve

File: content/blog/2022-1-18-January-22-heartbeat.md [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+
+<external-link 
+href="https://www.kdnuggets.com/2021/12/versioning-machine-learning-experiments-tracking.html"
+title="Versioning Machine Learning Experiments vs Tracking Them"
+description="Maria Khalusova's tutorial on DVC's `exp init` command and the next level of experiment tracking that delivers true reproducibiity."
```
Comment (MKhalusova) [inline content/blog/2022-1-18-January-22-heartbeat.md:264]: typo: reproducibiity -> reproducibility

File: content/blog/2022-1-18-January-22-heartbeat.md [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+## New Team Members
+
+We have a few new team members this month!
+
+[**Daniele Trifirò**](https://github.com/dtrifiro) is our first team member from
+Italy! He joins us as a Senior Software Engineer. Daniele has a background in
+Physics/Astrophysics and worked for 4 years as a researcher in the LIGO
+Scientific collaboration and then went on to positions at Cloudian and illimity.
+It was at illimity where he "fell in love" with DVC! In his free time Daniele
+likes to listening to and sometimes playing music himself, as well as rock
```
Comment (MKhalusova) [inline content/blog/2022-1-18-January-22-heartbeat.md:277]: likes to listening to and  -> likes listening to and

File: content/blog/2022-1-18-January-22-heartbeat.md [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+climbing. 🧗🏼‍♂️
+
+[**Thomas Kunwar**](https://github.com/yathomasi) is a software engineer joining
+the team from Nepal. He's been working as a fullstack developer specializing on
```
Comment (MKhalusova) [inline content/blog/2022-1-18-January-22-heartbeat.md:281]: specializing on -> specializing in

File: content/blog/2022-1-18-January-22-heartbeat.md (hunk 1 above)
Comment (jendefig) [reply content/blog/2022-1-18-January-22-heartbeat.md:65]: Thanks Roger!

**Review by MKhalusova — CHANGES_REQUESTED** [review]
Found some typos

File: content/blog/2022-1-18-January-22-heartbeat.md [hunk 7]
```diff
@@ -0,0 +1,352 @@
...
+of his two part series,
+[**Diego Jardim**](https://www.linkedin.com/in/diegosevero/) of
+[Poatek](https://poatek.com/) takes us through the basics of MLOps and the
+stages of implementation and maturity of an MLOps pipeline. He closes by
+introducing us to some tools to help a team progress through these stages, which
+include DVC and CML.
+
+[In Part 2](https://poatek.com/2021/12/29/mlops-a-complete-and-hands-on-introduction-part-2/)
+he delves into more detail and code on how to set up version control of
+everything with DVC and automation of experimentation and reporting with CML.
```
[the remark this answers is not in the document]
Comment (jendefig) [reply content/blog/2022-1-18-January-22-heartbeat.md:59]: Yes.  That's better! 🙏

File: content/blog/2022-1-18-January-22-heartbeat.md [hunk 8]
```diff
@@ -0,0 +1,352 @@
...
+
+![MLOPs Tool evaluation process](/uploads/images/2022-01-18/ryan-dawson-thoughtworks-cd4ml.png '=800')
+_Ryan Dawson's MLOps tool evaluation process
+([Source link](https://www.thoughtworks.com/what-we-do/data-and-ai/cd4ml/guide-to-evaluating-mlops-platforms1?utm_source=linkedin&utm_medium=social-organic&utm_campaign=tw-webinars_2021-12&gh_src=463a2f181us))_
+
+[**Dean Pleban**,](https://www.linkedin.com/in/deanpleban/) CEO of
+[DAGsHub,](https://dagshub.com) also gave a great talk on a decision making
+framework for deciding on your tools in his presentation at
+[DevOpsDays Tel Aviv](https://devopsdays.org/events/2021-tel-aviv/welcome/). In
+this talk you will learn guidelines and mental models that will help you choose
```
[the remark this answers is not in the document]
Comment (jendefig) [reply content/blog/2022-1-18-January-22-heartbeat.md:172]: Not necessary here.

File: content/blog/2022-1-18-January-22-heartbeat.md (hunk 1 above)
Comment (jendefig) [reply content/blog/2022-1-18-January-22-heartbeat.md:65]: @rogermparent  I corrected this and it's not showing that it changed.  
image

File: content/blog/2022-1-18-January-22-heartbeat.md [hunk 9]
```diff
@@ -0,0 +1,352 @@
...
+  - Reproducibility
+  - Stateful training
+  - SQL
+  - Git
+---
+
+# From the Community
+
+Happy New Year! Hope you got some good R&R and stayed healthy at the end of
+2021, because 2022 has lots of great things in store!
```
[the remark this answers is not in the document]
Comment (jendefig) [reply content/blog/2022-1-18-January-22-heartbeat.md:43]: I think it needs the comma.

File: content/blog/2022-1-18-January-22-heartbeat.md (hunk 1 above)
Comment (jendefig) [reply content/blog/2022-1-18-January-22-heartbeat.md]: Nvrmnd - file problem.  Resolved.

**Comment by jendefig** [comment]
> Looks like Medium doesn't play well with the link checker! I guess we'll need some extra configuration like with linkedin profiles.

I attempted to fix the medium links, but still no good. 😞 Guess I need help here @rogermparent.

**Comment by rogermparent** [comment]
Hey @jendefig! Sorry about the link-check issues, it's not on your end- some sites don't take kindly to automated requests that aren't an actual visitor, and so we have to exclude them from the link check. This could go in as-is, but to prevent confusion in the future we can exclude medium links from the check.

**Comment by iesahin** [comment]
Read it. Good content. Thank you Jeny.

**Comment by jendefig** [comment]
@MKhalusova Can this go?  I just didn't agree with one comma.

**Comment by shcheklein** [comment]
@jendefig do we need the main image?

**Comment by jendefig** [comment]
> @jendefig do we need the main image?

Yes.  The problem is the the path needs to be corrected in the .md.  I have done it here locally, but accidentally pushed it to master instead of the branch and then attempted to roll it back with git revert and git reset HEAD.  But it still didn't work, so then I pulled and got Roger's Yaml file excluding the links in my computer and it wasn't showing my change, just the new file even though it is saved.  I still can get it right.  I've spent an hour trying to correct this.  And I'm done.  Can you please just change the path to this: 2022-01-18/heartbeat-january.png. The image there is correct.  Just the path is not.  And then this is ready to go.  Maria and I just disagreed on one comma in the first sentence.  She's done for the day.

**Comment by shcheklein** [comment]
Hey, sure. No worries, Git gives a lot of headaches to everyone :)

I would recommend to use GitHub in simple cases like this:

Screen Shot 2022-01-18 at 4 54 45 PM

(note the Edit file button)

I've done editing. Let's wait for it to be deployed and I'll merge this.

**Review by shcheklein — APPROVED** [review]

**Comment by jendefig** [comment]
> Hey, sure. No worries, Git gives a lot of headaches to everyone :)
> 
> I would recommend to use GitHub in simple cases like this:
> 
> Screen Shot 2022-01-18 at 4 54 45 PM
> 
> (note the Edit file button)
> 
> I've done editing. Let's wait for it to be deployed and I'll merge this.

Ok.  Thanks @shcheklein! 🙏   I will try that way next time!

**Pull request merged**